tensorrt_llm/_torch/pyexecutor/_util.py

```
@@ -925,6 +1210,53 @@ def _create_kv_cache_manager(
+                isinstance(raw_dtype_spec, dict)
+                or ":" in str(raw_dtype_spec)):
+            per_layer_dtype_map = _resolve_per_layer_dtype(
+                raw_dtype_spec, num_hidden_layers, quant_config, kv_cache_dtype)
+            unique_dtypes = set(per_layer_dtype_map.values())
+            if len(unique_dtypes) > 1:
+                # Build per_layer_num_kv_heads before delegating
+                draft_config_for_kv_early = None
+                if layer_mask is None:
+                    draft_config_for_kv_early = (getattr(
+                        model_engine.model, 'draft_config', None)
+                                                 if model_engine is not None
```

Flake8 (7.3.0)

[error] continuation line over-indented for hanging indent

(E126)